Dockerfile.client

```diff
@@ -88,6 +88,7 @@ RUN mkdir -p /tmp/client/bin && \
     cp src/clients/python/ensemble_image_client.py /tmp/client/python/. && \
     cp src/clients/python/grpc_image_client.py /tmp/client/python/. && \
     cp src/clients/python/simple_client.py /tmp/client/python/. && \
+    cp src/clients/python/multi_port_client.py /tmp/client/python/. && \
```

Comment:
src/clients/... is not for test clients. It is for clients that have some more general use or that are used for examples. So just rename it to multi_port_test.py and put it in L0_multiple_ports